gapis/gfxapi/vulkan/dead_code_elimination.go

```diff
@@ -0,0 +1,254 @@
+// Copyright (C) 2017 Google Inc.
```

Comment:
This file appears to be almost identical to gles/dead_code_elimination.go. Other than a bit of different logging, I cannot find any differences.

Is it possible to move them both into a common place, and not duplicate?